docs/UserGuide.md

```diff
@@ -301,13 +302,27 @@ Examples:
 * `re_edit 1 1 d/Tuesday`
 * `re_edit 1 1 t/14:00`
 
+### Export a person: `export`
+Export a person's details from the address book. 
+
+Format: `export [INDEX_OF_PERSON]`
+
+* Exports a person's details with the specified `INDEX`
+* All details except groups and tags are exported.
+* Export data is saved in the data/export.json
+* Exporting a person will overwrite any existing export.json file
+
+Examples:
+* `export 1`
+* `export 2`
+
 ### Find free time slots: `free`
 
 Displays the time slots in a week when all members of the specified group are free (no events).
 
-Format: free GROUP_INDEX [START_DATE]
+Format: `free GROUP_INDEX [START_DATE]`
 
-* Finds unoccupied time slots within members of the group at the specified `INDEX`. The index refers to the index number
+* Finds unoccupied time slots within members of the group at the specified `GROUP_INDEX`. The index refers to the index number
   shown in the displayed group list. The index *must be a positive* 1, 2, 3 ...
 * The `START_DATE` field is optional. The time slots will be shown for the week ahead, starting from that date. If it is
```

Comment:
Perhaps the format of `START_DATE` can be mentioned here `dd/MM/yyyy` cause the previous `START_DATE` consist of time